# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../newt/compile-fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur newt-0.51.6.orig/snackmodule.c newt-0.51.6/snackmodule.c
--- newt-0.51.6.orig/snackmodule.c	2003-02-05 09:11:35.000000000 +0200
+++ newt-0.51.6/snackmodule.c	2005-09-15 16:51:33.000000000 +0300
@@ -11,6 +11,8 @@
 #include <sys/time.h>
 #include <unistd.h>
 
+#undef _POSIX_C_SOURCE
+
 #include "Python.h"
 #include "newt.h"
 #include "newt_pr.h"
